src/components/NewsList.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const NewsContent = (props) => {
+  const numEntries = props.numEntries
+    ? props.numEntries
+    : props.articles.length;
+  const colorStyle = (index) => ({
+    color: index % 2 == 0 ? "var(--color-accent-eng)" : "var(--color-accent-math)",
+  });
+
+  return props.articles.splice(0, numEntries).map((article, i) => (
+    <tr>
+      <th>
+        <a href={article.link}>
```

Comment:
if this is an internal link, use react router

Reply:
(separate PR if it doesnt exist yet)

Reply:
I'm not sure if the news will be an internal link but I've changed all the others